### PR TITLE
Terminal: Fix some missing text attributes

### PR DESCRIPTION
**Pull request by rburchell**
Probably doesn't actually change much yet since we don't support many
text rendering options, but it's at least good to have the options, and
to record things we don't yet support too.

### CODE REVIEW

File: Applications/Terminal/Terminal.cpp [hunk 1]
```diff
@@ -180,8 +214,16 @@ void Terminal::escape$m(const ParamVector& params)
         case 46:
         case 47:
             // Background color
+            if (m_current_attribute.flags & Attribute::Bold)
+                param += 8;
             m_current_attribute.background_color = param - 40;
             break;
+        case 49:
+            // reset foreground
```
Comment (rburchell) [inline Applications/Terminal/Terminal.cpp]: This should be 'background', my mistake.

**Comment by christopherdumas** [comment]
This shouldn't mess up anything I'm doing. But I'm going to end up rewriting the painting code (that seems to be what's failing), so I'm not sure it'd be a good idea to actually make this work before I merge my PR, that seems merge-conflict-tastic.

**Comment by rburchell** [comment]
I don't have any intention of changing painting anytime soon, don't worry. Also, it would need some more extensive work in SharedGraphics first, as it doesn't seem like we have any support for text attribute rendering at all yet.

**Comment by christopherdumas** [comment]
And I thought the Terminal would be the simplest GUI app!

**Comment by rburchell** [comment]
@Mustafa32 I'd be interested to see if this improves the terminal color stuff you were having trouble with at all.

**Pull request merged**